pages/dkp/kaptain/2.0.0/install/dkp/index.md

```diff
@@ -141,46 +151,54 @@ Use the following instructions to install Spark Operator from Kommander Catalog
   ```bash
   kubectl kudo plan status --instance kaptain -n kubeflow
   ```
+### Enable Kaptain with a custom configuration
+
+<!-- Is this an alternative or an additional step to the section above: Create a Git repository for Kaptain? -->
```

Comment:
yeah this could use some clarifications why customers would want to do it.

Reply:
after reviewing #4339 I think this is duplicated and should only be in #4339 as it seems to be part of deployment. at least if I udnerstand things correctly. (cc @lukeogg to verify)

Reply:
To sum up, we should move lines 154 to 189 (### Enable Kaptain with a custom configuration) to the deployment page, correct? + correct, verify yaml lines to match what is needed for Kaptain.